tests/test_utils.py

```diff
@@ -1,35 +1,73 @@
-# import pytest
-# from agoradatatools.etl import utils
-# from synapseclient import Synapse
+import builtins
+import io
+from unittest.mock import patch
 
-# file_object = {
-#     "id": "syn25838546",
-#     "format": "table",
-#     "final_filename": "teams",
-#     "provenance": [],
-#     "destination": "syn25871921"
-# }
+import pytest
+import synapseclient
+import yaml
 
+from agoradatatools.etl import utils
 
-# def test_login():
-#     assert type(utils._login_to_synapse()) is Synapse
 
+def test_get_config_with_invalid_file_path():
+    with pytest.raises(FileNotFoundError, match="File not found. *"):
+        utils._get_config(config_path="this/is/a/bad/path")
 
-# def test_yaml():
-#     # tests if a valid file renders a list
-#     assert type(utils._get_config()) is list
 
-#     # tests if a bad file will
-#     with pytest.raises(SystemExit) as err:
-#         utils._get_config(config_path="./tests/test_assets/bad_config.yaml")
-#     assert err.type == SystemExit
-#     assert err.value.code == 9
+def test_get_config_with_parser_error():
+    with pytest.raises(
+        yaml.parser.ParserError, match="YAML file unable to be parsed. *"
+    ):
+        utils._get_config(config_path="./tests/test_assets/bad_config_parsing.yaml")
 
-#     with pytest.raises(SystemExit) as err:
-#         utils._get_config(config_path="./tests/test_assets/bad_config.yam")
-#     assert err.type == SystemExit
-#     assert err.value.code == 2
 
+def test_get_config_with_scanner_error():
+    with pytest.raises(
+        yaml.scanner.ScannerError, match="YAML file unable to be scanned. *"
+    ):
+        utils._get_config(config_path="./tests/test_assets/bad_config_scanning.yaml")
 
-# if __name__ == "__main__":
-#     pytest.main()
+
+def test_get_config_with_no_config_path():
```

Comment:
The tests seem to be failing because the previous mocks aren't being stopped.  Try adding 

```
mock.patch.stopall()
```

in the teardown for the `test_process.py` class.

Reply:
That worked. I will start adding this to all teardowns